src/components/notifications/notifications.service.ts

```diff
@@ -336,6 +356,56 @@ export function getSlackConfiguration(slackConfigId: number, isDeleteComponent?:
     })
 }
 
+export function getWebhookConfiguration(webhookConfigId: number): Promise<ResponseType> {
+    return get(`${Routes.NOTIFIER}/channel/webhook/${webhookConfigId}`)
+}
+
+export function saveWebhookConfiguration(data): Promise<UpdateConfigResponseType> {
```

Comment:
can we combined save and update method to reduce duplicacy?

Reply:
done